resources/js/components/dues/DuesRequiredInfo.vue

```diff
@@ -174,6 +196,46 @@ export default {
     localUser: function() {
       return this.user;
     },
+    selectedPackage: function() {
+      if (null === this.duesPackages) return null;
```

Comment:
You should enclose the body of the `if` statement in brackets so no one gets confused later